js/tabs.js

```diff
@@ -0,0 +1,34 @@
+class TabManager {
+    constructor() {
+        this.tabContainer = {}
+    };
+
+    create(tabData) {
```

Comment:
would help to comment on representative examples of when this is called, just for context